### PR TITLE
UXUI 583 - Map the UI Toolkit

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do? (please provide any background)
This maps the UI-Toolkit to our brands repo, it uses bootstrap based classes and variables.
This should eventually stop people used the UI-Toolkit CSS (which was originally designed as a stand alone open source css file)

There are a few components (i.e countdown) which do not relate to bootstrap classes, these will be added as we look at styling solutions in the future.

#### What tests does this PR have?
None

#### How can this be tested?
There is a an equivalent branch (UXUI-583) on the pattern library that you can test this on. You will need to run pattern library them NPM link to this branch on brands. Please note this is only in HX brand.

#### Screenshots / Screencast
screen shot 2016-04-20 at 14 45 48

#### What gif best describes how you feel about this work?
cffb4700-bab0-4a46-9cbe-c1ac2a7ae9c0
---

- [x] I have checked our general [contributing document](../CONTRIBUTING.md) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [x] :+1:

**Review 2** \*
- [x] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\* this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

**Comment by GeorgeF72** [comment]
1️⃣

**Comment by GeorgeF72** [comment]
👍

**Comment by jackdcrawford** [comment]
2️⃣ ✌️ 🔍

File: brands/globals/src/less/toolkit/_toolkit.less [hunk 1]
```diff
@@ -0,0 +1,15 @@
+@import '_colors';
+
+/* -- Atoms -- */
+@import 'components/_buttons';
+@import 'components/_flags';
+@import 'components/_image';
+@import 'components/_input';
+@import 'components/_losenge';
```
Comment (jackdcrawford) [inline brands/globals/src/less/toolkit/_toolkit.less]: ~~losenge~~**lozenge**, the file name needs the same change too

File: brands/globals/src/less/toolkit/components/_flags.less [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+.component-flag {
+  .flag-variant(@default);
+  color: @text-color;
+  &.default {
+    .flag-variant(@default);
+  }
+  &.primary {
+    .flag-variant(@primary);
+  }
+  // &.secondary {
```
Comment (jackdcrawford) [inline brands/globals/src/less/toolkit/components/_flags.less]: Can this be removed?

**Comment by jackdcrawford** [comment]
👍

**Pull request merged**